atsign-foundation/at_c
# chore: wrap all .h files with extern "C" macros

**- What I did**

- Added extern C macros to all header files in the project.

**- How I did it**

**- How to verify it**

- **Enable ignore whitespace for the review**
  > a bunch of those files were not following our standard formatting rules

- I checked to see that the macros were well formed, but I couldn't find any
  clear information as to what is allowed inside them... I'm not sure how the
  linker resolves includes and defines inside the extern block, nor do I have a
  C++ project to test.

**- Description for the changelog**
chore: wrap all .h files with extern "C" macros
